ambry-store/src/main/java/com.github.ambry.store/BlobStore.java

```diff
@@ -212,6 +212,7 @@ public void start() throws StoreException {
                 queueProcessingPeriodInMs, config.storeStatsWaitTimeoutInSecs, time, longLivedTaskScheduler,
                 taskScheduler, diskIOScheduler, metrics);
         checkCapacityAndUpdateWriteStatusDelegate(log.getCapacityInBytes(), index.getLogUsedCapacity());
+        logger.trace("The store: " + storeId + " is successfully started");
```

Comment:
use log formatting

`logger.trace("The store {} is successfully started", storeId);`